File: Kotlin/1043.kt
import java.util.Scanner

fun main() {
    val input = Scanner(System.`in`);
    val a = input.nextDouble();
    val b = input.nextDouble();
    val c = input.nextDouble();

    if(a < (b + c) && b < (c + a) && c < (a + b)){
        println("Perimetro = %.1f".format( (a + b + c)));
    }
    
    else{
        println("Area = %.1f".format((((a + b) * c) / 2)));
    }
}

File: Php/1043.php
<?php
[$a, $b, $c] = explode(" ", readline());
if ($a < ($b + $c) && $b < ($c + $a) && $c < ($a + $b)) {
    printf("Perimetro = %.1f\n", ($a + $b + $c));
}
else {
    printf("Area = %.1f\n", (($a + $b) * $c) / 2);
}
?>